.travis.yml

```diff
@@ -36,15 +36,15 @@ matrix:
           env: COVERAGE=true TEST_COMMAND="composer test-ci"
 
           # Test LTS versions
-        - php: 7.2
+        - php: 7.3
           env: DEPENDENCIES="dunglas/symfony-lock:^2"
-        - php: 7.2
+        - php: 7.3
           env: DEPENDENCIES="dunglas/symfony-lock:^3"
-        - php: 7.2
```

Comment:
Is this really needed? I know travis is quicker to start on 7.2.. Ah, anyhow... 

It is not needed but why not =)

Reply:
we can also wait with merging this until 7.3 is released as final, rather than RC. i guess then travis will also be quick with it.